### DISCUSSION TITLE
Why the requirement to pass an illuminant in the "colour.sd_to_XYZ" definition?

### DISCUSSION THREAD

**Discussion by hminle** [colour-science/colour#1034] [Q&A] [answered]
Hi Colour Team,

I have a question about how to convert spectral data to XYZ correctly.
In [the sd_to_XYZ function](https://colour.readthedocs.io/en/develop/generated/colour.sd_to_XYZ.html?highlight=sd_to_XYZ), why we need to specify `illuminant`?
As I read from many other sources, we do not need to do that, right?
Just like this:
<img width="437" alt="image" src="https://user-images.githubusercontent.com/16201681/186567247-17baee4e-7b1a-457d-882e-ed34865f946a.png">
Source: [color-matching-function](https://www.sciencedirect.com/topics/computer-science/color-matching-function)

Thanks

**Comment by thomasmansencal** [comment, marked as answer]
Hi @hminle,

If you look at the definition signature, it is an optional argument so it does not need to be passed and will default to *CIE Illuminant E*, equivalent to multiplying by 1.

Cheers,

Thomas

**Reply by hminle** [reply]
Oh thank you so much @KelSolaar . But for other cases, `that illuminant argument` is for viewing condition, right?
